### PR TITLE
test(fa): osk e2e test

### PR DESCRIPTION
**Pull request by Rikkiabb**
# ...
https://app.asana.com/0/1201498198362488/1201498198362470

## What
E2E for applicant flow in financial aid

## Why
Testing is niceeee

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] Formatting passes locally with my changes
- [X] I have rebased against main before asking for a review

### CODE REVIEW

File: workspace.json [hunk 1]
```diff
@@ -4145,12 +4140,10 @@
...
             "production": {
-              "fileReplacements": [
-                {
-                  "replace": "apps/financial-aid/web-osk/src/environments/environment.ts",
-                  "with": "apps/financial-aid/web-osk/src/environments/environment.prod.ts"
-                }
-              ]
+              "fileReplacements": [{
+                "replace": "apps/financial-aid/web-osk/src/environments/environment.ts",
+                "with": "apps/financial-aid/web-osk/src/environments/environment.prod.ts"
```
Comment (darrikonn) [inline workspace.json:4145]: I don't think that file replacements work with esbuild

**Review by darrikonn — COMMENTED** [review]
Need to verify my comment first.

File: apps/financial-aid/web-osk-e2e/src/support/commands.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+type KeyMapping<TKey extends string, TValue> = { [K in TKey]: TValue }
+
+const getFixtureFromRequest: KeyMapping<string, { fixture: string }> = {
+  CurrentUserQuery: { fixture: 'currentUserQuery' },
+  getNationalRegistryUserQuery: { fixture: 'nationalRegistryUserQuery' },
+  GetMunicipalityQuery: { fixture: 'municipalityQuery' },
+  createApplication: { fixture: 'createApplicationMutation' },
+}
+
```
Comment (MargretFinnboga) [inline apps/financial-aid/web-osk-e2e/src/support/commands.ts:21]: 👏 👏

File: apps/financial-aid/web-osk/src/routes/application/HomeCircumstancesForm/homeCircumstancesForm.tsx [hunk 3]
```diff
@@ -86,43 +86,48 @@ const HomeCircumstancesForm = () => {
...
-                homeCircumstancesCustom: event.target.value,
-              })
-            }}
-          />
-        </Box>
+        {hasError && !form?.homeCircumstances && (
+          <div
+            data-testid="errorMessage"
+            className={cn({
+              [`errorMessage`]: true,
```
Comment (MargretFinnboga) [inline apps/financial-aid/web-osk/src/routes/application/HomeCircumstancesForm/homeCircumstancesForm.tsx:93]: check if this effects animation ? 🎭

**Review by MargretFinnboga — COMMENTED** [review]
👀

File: apps/financial-aid/web-osk/src/routes/application/HomeCircumstancesForm/homeCircumstancesForm.tsx (hunk 3 above)
Comment (Rikkiabb) [reply apps/financial-aid/web-osk/src/routes/application/HomeCircumstancesForm/homeCircumstancesForm.tsx:93]: Checked and it looks good, but will add as a note for QA 🧪

**Review by MargretFinnboga — APPROVED** [review]
Testing never looked so good 😎

**Pull request merged**